.github/workflows/tests.yml

```diff
@@ -9,8 +9,12 @@ on:
       - staging
       - main
 
+env:
+  MEILISEARCH_URL: http://localhost:7700
+
 jobs:
   integration-tests:
+    needs: linter
```

Comment:
Are you sure you don't want to run the tests even if your linting failed? Sometimes you wait until the end to fix some very annoying linting errors.

Reply:
I tried to follow an example I saw with the platform tests, but it didn't work.